src/util.jl

```diff
@@ -0,0 +1,40 @@
+# LibFTD2XX.jl - Utility Module
+#
+# By Reuben Hill 2019, Gowerlabs Ltd, reuben@gowerlabs.co.uk
+#
+# Copyright (c) Gowerlabs Ltd.
+
+module Util
+
+export ntuple2string
+
+using Compat
+
+"""
+    ntuple2string(input::NTuple{N, Cchar} where N)
+
+Convert an NTuple of Cchars (optionally null terminated) to a julia string.
+
+# Example
+
+```jldoctest
+julia> ntuple2string(Cchar.(('h','e','l','l','o')))
+"hello"
+
+julia> ntuple2string(Cchar.(('h','e','l','l','o', '\0', 'x'))) # null terminated
+"hello"
+```
+
+"""
+function ntuple2string(input::NTuple{N, Cchar} where N)
```

Comment:
Is it nor normal to write `ntuple2string(input) where {N}`, e.g the `where` is outside the brackets, check style guide

Reply:
fixed in 913629eab9961a4149e5b7ae84baf2b54d9fcd2f